create_data_image.sh

```diff
@@ -33,9 +33,7 @@ IMAGE_SIZE=$((8 + ${TOTAL_SIZE} / (920*1024)))
 echo "Creating data partition of ${IMAGE_SIZE} MB..."
 mkdir -p images
 dd if=/dev/zero of=images/data.bin bs=1M count=${IMAGE_SIZE}
-MKE2FS_CONFIG=mke2fs.conf /sbin/mkfs.ext4 -b4096 -I128 -m3 \
-		-E lazy_itable_init=0,lazy_journal_init=0,resize=268435456 \
-		-F images/data.bin
+MKE2FS_CONFIG=mke2fs.conf /sbin/mke2fs -t od-data -m3 -F images/data.bin
 echo
```

Comment:
The `-m3` argument can also be moved into the config file, as `reserved_ratio`.

I'm actually wondering if 3% is still too much. We don't actually need that much space for root, since we log into tmpfs. But the man page does mention that it also helps reduce fragmentation, so setting it to zero may not be ideal either.

Reply:
Done